config/routes.rb

```diff
@@ -22,7 +22,13 @@
         get "historic" => "activities#historic"
       end
     end
-    resources :organisations, except: [:destroy] do
+    resources :organisations, except: [:destroy, :index] do
+      constraints role: /delivery_partners|matched_effort_providers/o do
+        collection do
+          get "(:role)", to: "organisations#index", defaults: {role: "delivery_partners"}, as: ""
```

Comment:
The commit message mentions the `nulldb` but I am not seeing and I thought we got rid of that - am I missing a it?

Reply:
Nope, I just forgot the edit the commit message. Have binned it.